static/js/app.js

```diff
@@ -28,15 +28,18 @@ async function updateEmailForwardingPrefs(submitEvent) {
     const response = await sendForm(forwardingPrefForm.action, formData);
     if (response && response.status === 200) {
       prefToggle.classList.toggle("forwarding-disabled");
-      prefToggle.title = prefToggle.dataset.defaultTitle;
       if (prefToggle.value === "Enable") {
+        prefToggle.title = prefToggle.dataset.defaultBlockingTitle;
         toggleLabel.textContent = toggleLabel.dataset.defaultForwardingLabel;
         wrappingEmailCard.classList.add("is-enabled");
-        return prefToggle.value = "Disable";
```

Comment:
It took me soooo long to figure this out, because I'd completely missed there was an assignment (`=`) in the return statement :cry:

Reply:
Oof. Glad you found it!